vessel-runtime/src/main/java/com/textnow/android/vessel/VesselCache.kt

```diff
@@ -0,0 +1,28 @@
+package com.textnow.android.vessel
+
+interface VesselCache {
+    /**
+     * Get the current size of the cache
+     */
+    val size: Int
```

Comment:
What is size here? Is it the cache capacity or the number of items currently cached? Does this represent the number of bytes or objects?